3party/protobuf/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@ set(SRC
   protobuf/src/google/protobuf/io/zero_copy_stream_impl_lite.cc
   protobuf/src/google/protobuf/message_lite.cc
   protobuf/src/google/protobuf/repeated_field.cc
-  protobuf/src/google/protobuf/stubs/atomicops_internals_x86_gcc.cc
```

Comment:
nit: Generator expression may be cleaner here too.

Reply:
I really doubt it.